src/main/java/com/Acrobot/ChestShop/Listeners/ItemInfoListener.java

```diff
@@ -30,93 +30,108 @@
+        }
+    }
+    
+    @EventHandler(ignoreCancelled = true)
+    public static void addRepairCost (ItemInfoEvent event) {
         ItemMeta meta = event.getItem().getItemMeta();
         if (meta instanceof Repairable && ((Repairable) meta).getRepairCost() > 0) {
             iteminfo_repaircost.send(event.getSender(), "cost", String.valueOf(((Repairable) meta).getRepairCost()));
         }
     }
-
-    @EventHandler
-    public static void addEnchantment(ItemInfoEvent event) {
+    
+    @EventHandler(ignoreCancelled = true)
+    public static void addEnchantment (ItemInfoEvent event) {
         ItemStack item = event.getItem();
         ItemMeta meta = item.getItemMeta();
         CommandSender sender = event.getSender();
-
+        
         for (Map.Entry<Enchantment, Integer> enchantment : item.getEnchantments().entrySet()) {
             sender.sendMessage(ChatColor.AQUA + capitalizeFirstLetter(enchantment.getKey().getName(), '_') + ' ' + toRoman(enchantment.getValue()));
         }
-
+        
         if (meta instanceof EnchantmentStorageMeta) {
             for (Map.Entry<Enchantment, Integer> enchantment : ((EnchantmentStorageMeta) meta).getStoredEnchants().entrySet()) {
                 sender.sendMessage(ChatColor.YELLOW + capitalizeFirstLetter(enchantment.getKey().getName(), '_') + ' ' + toRoman(enchantment.getValue()));
             }
         }
     }
-
-    @EventHandler
-    public static void addPotionInfo(ItemInfoEvent event) {
+    
+    @EventHandler(ignoreCancelled = true)
+    public static void addPotionInfo (ItemInfoEvent event) {
         ItemStack item = event.getItem();
-
+        
         if (item.getType() != Material.POTION || item.getDurability() == 0) {
             return;
         }
-
+        
         Potion potion;
-
+        
         try {
             potion = Potion.fromItemStack(item);
         } catch (IllegalArgumentException ex) {
             return;
         }
-
+        
         StringBuilder message = new StringBuilder(50);
-
+        
         message.append(ChatColor.GRAY);
-
+        
         if (potion.getType() == null) {
             return;
         }
-
+        
         if (potion.isSplash()) {
             message.append("Splash ");
         }
-
+        
         message.append("Potion of ");
         message.append(capitalizeFirstLetter(potion.getType().name(), '_')).append(' ');
         message.append(toRoman(potion.getLevel()));
-
+        
         CommandSender sender = event.getSender();
-
+        
         sender.sendMessage(message.toString());
-
+        
         for (PotionEffect effect : potion.getEffects()) {
             sender.sendMessage(ChatColor.DARK_GRAY + capitalizeFirstLetter(effect.getType().getName(), '_') + ' ' + toTime(effect.getDuration() / 20));
         }
     }
-
-    @EventHandler
-    public static void addBookInfo(ItemInfoEvent event) {
+    
```

Comment:
Please revert all of those white space changes. Keeping the diff small helps keep a clean git history of who touched which files and a general similar code style.